index.js

```diff
@@ -194,9 +201,13 @@ yargs
   .command('cut [level]', 'cuts a release branch from origin/master', (yargs) => {
     yargs
       .positional('level', {
-        choices: ['major', 'minor'],
+        choices: ['major', 'minor', 'premajor', 'preminor', 'prerelease'],
```

Comment:
Are `premajor` and `preminor` documented? I can't seem to find info on them. I was wondering because it seems a little unnecessary to me to say I want a `prefoobar` type release _and_ also provide a `preid`. Maybe we could keep this as it was then select `premajor` or `preminor` based on if a `preid` is present?

Reply:
You can find documentation [for premajor and preminor in NPM versioning docs](https://docs.npmjs.com/cli/version#synopsis). Acceptable arguments are: 
```
npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease | from-git]
```

Preid is separate from the version itself. In npm, if you version to `premajor` the output will not have a preid. The preid makes it easy for us to identify and separate candidate releases. e.g. if no preid is passed, and you bump `0.1.0` to `premajor`, the version output will be `1.0.0-0`. The preid allows you to define what that pre patch is, e.g. `1.0.0-beta.0`. This is helpful for consumers of this module to define their prerelease how they like, e.g. `1.0.0-rc.0`, `1.0.0-alpha.0` etc.

Reply:
Basically, I figured we should follow semver and `npm version`, which don't care if you provide preid or not.